doc/api/stream.md

```diff
@@ -1480,6 +1480,31 @@ async function run() {
 run().catch(console.error);
 ```
 
+### Readable.from(iterable, [options])
+
+* `iterable` {Iterable} Object implementing the `Symbol.asyncIterator` or
+  `Symbol.iterator` iterable protocol.
+* `options` {Object} Options provided to `new stream.Readable([options])`.
+  By default, Readable.from will set `options.objectMode` to `true`, unless
```

Comment:
```suggestion
  By default, `Readable.from()` will set `options.objectMode` to `true`, unless
```

Reply:
Thanks I wasn't sure what the pattern here was!